<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedFormatWidget</class>
 <widget class="QWidget" name="AdvancedFormatWidget">
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fileNameRegexLabel">
     <property name="text">
      <string>File Name Regex</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="fileNameRegexLineEdit">
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="regexGroupsLabel">
     <property name="text">
      <string>Regex Group Names</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="tomviz::RegexGroupsWidget" name="regexGroupsWidget" native="true">
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="regexGroupsSubstitutionsLabel">
     <property name="text">
      <string>Group Substitutions</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="tomviz::RegexGroupsSubstitutionsWidget" name="regexGroupsSubstitutionsWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomviz::RegexGroupsWidget</class>
   <extends>QWidget</extends>
   <header location="global">RegexGroupsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomviz::RegexGroupsSubstitutionsWidget</class>
   <extends>QWidget</extends>
   <header>RegexGroupsSubstitutionsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
